includes/publisher/js/src/components/MediaUploader.jsx

```diff
@@ -140,6 +211,336 @@ export default function MediaUploader({ modelSlug, field, required }) {
 		});
 	}
 
+	/**
+	 * Click handler to use wp media uploader for repeater
+	 * @param e - event
+	 */
+	function multiClickHandler(e) {
+		e.preventDefault();
+
+		let library = {
+			order: "DESC",
+			orderby: "date",
+		};
+
+		if (allowedTypes) {
+			library.type = getAllowedTypesLongExtension();
+		}
+
+		if (field?.isFeatured) {
+			library.type = ["image"];
+		}
+
+		const getMediaModalTitle = () => {
+			const title = getMediaButtonText(field);
+			if (allowedTypes) {
+				return `${title} (${getAllowedTypesForUi().toUpperCase()})`;
+			}
+
+			return title;
+		};
+
+		// If the media frame already exists, reopen it.
+		if (media) {
+			media.open();
+			return;
+		}
+
+		const selectedIds = fieldValues.map((item) => {
+			return item.id;
+		});
+
+		const media = wp.media({
+			title: getMediaModalTitle(),
+			multiple: "add",
+			frame: "select",
+			library: library,
+			selected: selectedIds,
+			button: {
+				text: __("Done", "atlas-content-modeler"),
+			},
+		});
+
+		media.on("open", function () {
+			let selection = wp.media.frame.state().get("selection");
+
+			selectedIds.forEach(function (id) {
+				let attachment = wp.media.attachment(id);
+				selection.add(attachment ? [attachment] : []);
+			});
+		});
+
+		media.on("select", function () {
+			const imgArr = [];
+			media
+				.state()
+				.get("selection")
+				.map(function (attachment) {
+					imgArr.push({
+						id: attachment.attributes.id,
+						url: attachment.attributes.url,
+					});
+				});
+			setValues(imgArr);
+		});
+
+		media.open();
+	}
+
+	if (field.isRepeatableMedia) {
+		return (
+			<>
+				<div className={"field"}>
+					<label
+						htmlFor={`atlas-content-modeler[${modelSlug}][${field.slug}]`}
+					>
+						{field.name}
+					</label>
+					{field?.required && <p className="required">*Required</p>}
```

Comment:
The "Required" text should be translatable. Something like this (eyeballing this):
```suggestion
					{field?.required && <p className="required">*{__("Required", "atlas-content-modeler")}</p>}
```